include/aspect/material_model/rheology/visco_plastic.h

```diff
@@ -219,9 +219,10 @@ namespace aspect
           /**
            * Minimum and maximum viscosities used to improve the
            * stability of the rheology model.
+           * They are either single values or vectors of phases.
```

Comment:
```suggestion
           * These parameters contain one value per composition and phase (potentially the same value).
```
While you are changing this anyway, can you rename the parameters to `minimum_viscosity` and `maximum_viscosity`? This makes it easier to associate the input parameter in the parameter file with the variable in the code.